test/ModalTriggerSpec.js

```diff
@@ -101,4 +101,19 @@ describe('ModalTrigger', function() {
 
     contextSpy.calledWith('value').should.be.true;
   });
+
+  it('Should pass children refs through. Prevent future regressions.', function () {
+    let TestComponent = React.createClass({
```

Comment:
Both this and `instance` could be `const` if desired, just pointing it out. :stuck_out_tongue: